target/linux/kirkwood/base-files/etc/board.d/01_leds

```diff
@@ -34,6 +34,10 @@ case "$board" in
 	ucidef_set_led_default "health" "health" "pogo_e02:green:health" "1"
 	ucidef_set_led_default "fault" "fault" "pogo_e02:orange:fault" "1"
 	;;
+"pogo_v4")
```

Comment:
Drop the LEDs here. One of them is handled by diag.sh and I fail to see why the other need to be lit up after boot.